src/pollers/gateway.rs

```diff
@@ -79,6 +87,50 @@ pub struct ClientTlsConfig {
     pub client_private_key: Vec<u8>,
 }
 
+#[derive(Clone, Debug)]
+pub struct RetryConfig {
+    /// initial wait time before the first retry.
+    pub initial_interval: Duration,
+    /// randomization jitter that is added or subtracted from the interval length.
```

Comment:
Should explicitly state that this is used as a multiplier

Reply:
Done